doc/manual/release-notes/rl-2.1.xml

```diff
@@ -49,8 +49,38 @@ new features:</para>
     <varname>nix-support/propagated-user-env-packages</varname>.</para>
   </listitem>
 
+  <listitem>
+    <para>The Nix installer will no longer default to the Multi-User
+    installation for macOS. You can still <link
+    linkend="sect-multi-user-installation">instruct the installer to
+    run in multi-user mode</link>.
+    </para>
+  </listitem>
+
+  <listitem>
+    <para>The Nix installer now supports performing a Multi-User
+    installation for Linux computers which are running systemd. You
+    can <link
+    linkend="sect-multi-user-installation">select a Multi-User installation</link> by passing the
+    <option>--daemon</option> flag to the installer: <command>sh &lt;(curl
+    https://nixos.org/nix/install) --daemon</command>.
+    </para>
+
+    <para>The multi-user installer cannot handle systems with SELinux.
+    If your system has SELinux enabled, you can <link
+    linkend="sect-single-user-installation">force the installer to run
+    in single-user mode</link>.</para>
+  </listitem>
+
 </itemizedlist>
 
+<para>
+  Multi-user Nix users on macOS can upgrade Nix by running
+  <command>sudo -i sh -c 'nix-channel --update &amp;&amp; nix-env -iA
+  nixpkgs.nix'; sudo launchctl stop org.nixos.nix-daemon; sudo
+  launchctl start org.nixos.nix-daemon</command>.
+</para>
```

Comment:
I think the release notes are not the best place for platform-specific upgrade instructions, which are presumably the same for every release. Might be better to refer to some "Upgrading Nix" section.

Reply:
Fixed!